### DISCUSSION TITLE
Ente Photos Sync deletion

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#1351] [Enhancements]
I would love to have some "deletion" sync on android. Currently i always need to "accept" the delete prompt if i delete something on ente android app for both/device. Would be nice if the app on its own could just press the accept somehow. Also would be nice to have some kind of deletion sync from the webpage. Currently if i delete something from the page, it doesnt reflect on my device which is kinda meh. Sorting picutres is way more easy on the webpage but since it doesnt sync to my phone its a bummer.

**Comment by vishnukvmd** [comment]
Hey, both Android and iOS do not allow apps to automatically delete items from disk due to security concerns. Explicit user action is required.

What we could do is show a dialog when we detect that items were deleted on a different device, giving you an option to delete those items from your device as well in a single-click.

**Reply by ghost** [reply]
This would be a start and nice to have.

Second isn't it possible to somehow use device manager permissions or something to "auto accept" the confirmation window?

**Reply by vishnukvmd** [reply]
Not possible on iOS, and could land us in trouble on Android. Best for us to follow platform specifications.

**Reply by ghost** [reply]
so we can atleast get the "delete confirmation window if there was a deletion on the web/cloud", right?

**Reply by vishnukvmd** [reply]
Yes, that is possible.

**Comment by Magneticdud** [comment]
You (as an user) can remove the second delete confirmation that the android operating system is doing by giving extra permissions

The guide: https://www.howtogeek.com/783113/how-to-stop-google-photos-from-asking-permission-to-delete-photos/

I don't know if an app can detect this and prompt the user by directly opening that page to add the permission
